lib/tasks/upgrade.rake

```diff
@@ -0,0 +1,17 @@
+namespace :upgrade do
+  desc "Run sidekiq with old queues so it can finish deferred jobs"
+  task :run_legacy_queues do
+    system "bundle exec sidekiq --daemon -q maintenance -q dispatch -q delete_account -q http -q http_service"\
+           " -q export -q photos -q socket_webfinger -q mail -q receive_local -q receive -q receive_salmon"\
```

Comment:
> We can't downgrade, so being able to just tell them to run this task to recover as much as possible is extremely helpful.

But 95% of the queued jobs probably are the old send/receive jobs, which wouldn't work anymore. So we can also tell people to run `bundle exec sidekiq -q ...` to recover if they didn't do that before? But most people should follow the guide and run sidekiq before the pull.

But since this task still is useful for people currently running develop, we should probably keep it, but I'd keep it as simple as possible: start sidekiq and Ctrl-C.